doc/OnlineDocs/contributed_packages/parmest/driver.rst

```diff
@@ -156,3 +156,16 @@ expression which is used to define "SecondStageCost".  The objective
 function can be used to customize data points and weights that are used
 in parameter estimation.
 
+Suggested initialization procedure for parameter estimation problems
+--------------------------------------------------------------------
+
+To check the quality of initial guess values provided for the fitted parameters, we suggest solving a 
+square instance of the problem prior to solving the parameter estimation problem using the following steps:
+
+1. Create :class:`~pyomo.contrib.parmest.parmest.Estimator` object
+
+2. Call :class:`~pyomo.contrib.parmest.parmest.Estimator.objective_at_theta` with optional argument ``(initialize_parmest_model=True)``. Different initial guess values for the fitted parameters can be provided using optional argument `theta_values` (**Pandas Dataframe**)
+
+3. (Optional) Initialize parameter estimation solve from the square problem solution by setting IPOPT solver option ``bound_push`` to a value of ``1e-8``
```

Comment:
This step would need to be done when the Estimator object is created.  I suggest moving this step into step 1 and formatting the optional input as a dictionary `solver_options={'bound_push': 1e-8}`